### DISCUSSION TITLE
汽車駕照

### DISCUSSION THREAD

**Discussion by seastar0687** [1101-GS4719-C/Python-Community-GS4719-C#3704] [General]
上了大學之後，許多朋友都跑去報駕訓班了，然後拿到汽車駕照了
而且中央大學正下方就有一個駕訓班，對報名駕訓班震得很方便
我自己也在前陣子報名駕訓班，並成功拿到駕照了
而且對於駕照我覺得要考究一次考到位，所以我是報考手排車
但身邊的人都覺得其實現在根本開不到手排車，報名自排就好

大家又是怎麼想的呢

**Comment by kai1660** [comment]
考手牌是明智的選擇，這樣兩種車都會開，有臨時要開手排車的情況，就不用怕了。

**Comment by kai1660** [comment]
考手牌是明智的選擇，這樣兩種車都會開，有臨時要開手排車的情況，就不用怕了，假如之後有需要開貨車或是超跑，通常這一的車都是手排，採離合放煞車，這樣的技能不可少，就去學手排吧。
